docs/manifest.json

```diff
@@ -321,7 +321,7 @@
 									"path": "./admin/templates/managing-templates/change-management.md"
 								},
 								{
-									"title": "Devcontainers",
+									"title": "Development containers on Coder",
```

Comment:
Just a thought: do we need to mention 'Coder' in the Quickstart and Dev Containers docs sidebar? Since users already know they're on Coder docs, it feels a bit redundant and might flow more naturally without it

Reply:
potential SEO, I think - but I actually don't know if it days anything / is worth doing (cc @mattvollmer - does it make sense to have `Coder` in the title or heading in 2024?)